<launch>

    <!--- Run Rviz 运行rivz-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find map_wise)/config/rviz/calib.rviz" />

    <!--- 运行激光雷达校准算法-->
    <node pkg="map_wise" type="lidar_calib_node" name="lidar_calib_node" output="screen" respawn="false">
        <param name="package_name" type="string" value = "map_wise" />
    </node>


</launch>